releases/UWP/net-native1.7/README.md

```diff
@@ -1,15 +1,18 @@
-# .NET Native 1.7 (UWP 5.4.x) Release Notes
+# .NET native tools 1.7 Release Notes
 
 You can see what was included in each .NET Native 1.7 ([Microsoft.NETCore.UniversalWindowsPlatform 5.4.x](https://www.nuget.org/packages/Microsoft.NETCore.UniversalWindowsPlatform)) release, below.
 
 When using Visual Studio these packages require Visual Studio 2017 or later.
```

Comment:
Can they be used outside of Visual Studio? 
If not, just "These packages require..."
If so, "If used with Visual Studio, these packages..."
Same comment applies to line 5 of the next document.